public/app/plugins/panel/singlestat/module.ts

```diff
@@ -243,47 +244,6 @@ class SingleStatCtrl extends MetricsPanelCtrl {
     this.render();
   }
 
-  getDecimalsForValue(value) {
-    if (_.isNumber(this.panel.decimals)) {
```

Comment:
So if user specified manually a decimals amount (panel.decimals), the auto decimal logic is ignored, this logic seems to have gotten lost, or does manual decimals still work?